docs/configuring-a-resource.md

```diff
@@ -557,13 +559,66 @@ p.AddResourceConfigurator("aws_autoscaling_group", func(r *config.Resource) {
 })
 ```
 
+### Initializers
+
+Initializers establish ownership of the supplied Managed resources in Crossplane.
+This typically involves the operations that are run before calling any
+ExternalClient methods.
```

Comment:
Seems like the definition is taken from runtime but I think it's wrong; there is no ownership operation done in initializer. I think scoping it as things that are done before the reconciliation begins would be more appropriate. Changing the comment in runtime as well would be good, too.